Orange/widgets/data/owpaintdata.py

```diff
@@ -905,18 +905,18 @@ def _init_ui(self):
             indBox, self, "brushRadius", minValue=1, maxValue=100,
             createLabel=False
         )
-        form.addRow("Radius", slider)
+        form.addRow("Radius:", slider)
 
         slider = gui.hSlider(
             indBox, self, "density", None, minValue=1, maxValue=100,
             createLabel=False
         )
 
-        form.addRow("Intensity", slider)
+        form.addRow("Intensity:", slider)
 
         gui.rubber(self.controlArea)
         gui.auto_commit(self.left_side, self, "autocommit",
-                        "Send", "Send on change")
+                        "Send")
```

Comment:
```py
label.title().capitalize() + ' on change'
```

How about _Apply_ as default instead of _Send_ or _Commit_ or overriding everywhere?